torchani/aev.py

```diff
@@ -121,8 +117,8 @@ def compute_shifts(cell, pbc, cutoff):
     ])
```

Comment:
There is a bug in JIT for `torch.arange` behaving differently from the eager mode. We should manually specify `dtype` as `torch.long` here.
See: https://github.com/pytorch/pytorch/issues/17662